qiskit_ibm_runtime/options/estimator_options.py

```diff
@@ -12,101 +12,108 @@
 
 """Estimator options."""
 
-from typing import Union, Literal
+from typing import Union
 
-from pydantic import Field, field_validator
+from pydantic import Field
 
 from .utils import (
     Dict,
     Unset,
     UnsetType,
-    skip_unset_validation,
 )
 from .execution_options import ExecutionOptionsV2
 from .resilience_options import ResilienceOptionsV2
 from .twirling_options import TwirlingOptions
+from .dynamical_decoupling_options import DynamicalDecouplingOptions
 from .options import OptionsV2
-from .utils import primitive_dataclass
+from .utils import primitive_dataclass, make_constraint_validator
 
-DDSequenceType = Literal["XX", "XpXm", "XY4"]
 MAX_RESILIENCE_LEVEL: int = 2
 MAX_OPTIMIZATION_LEVEL: int = 1
 
 
 @primitive_dataclass
 class EstimatorOptions(OptionsV2):
-    """Options for EstimatorV2.
+    """Options for V2 Estimator.
 
     Args:
+        default_precision: The default precision to use for any PUB or ``run()``
+            call that does not specify one.
+            Each estimator pub can specify its own precision. If the ``run()`` method
+            is given a precision, then that value is used for all PUBs in the ``run()``
+            call that do not specify their own. Default: 0.02.
+
+        default_shots: The total number of shots to use per circuit per configuration.
+
+            .. note::
+                If set, this value overrides :attr:`~default_precision`.
+
+            A configuration is a combination of a specific parameter value binding set and a
+            physical measurement basis. A physical measurement basis groups together some
+            collection of qubit-wise commuting observables for some specific circuit/parameter
+            value set to create a single measurement with basis rotations that is inserted into
+            hardware executions.
+
+            If twirling is enabled, the value of this option will be divided over circuit,
+            randomizations, with a smaller number of shots per randomization. See the
+            :attr:`~twirling` options.
+
+
         optimization_level: How much optimization to perform on the circuits.
```

Comment:
Is optimization_level current used for anything? If not I suggest we remove it and only add it back later when it is used for something. If it is used for something it should be documented as something more specific than "light optimization"